### PR TITLE
Added setup to send otp mails

### PR DESCRIPTION
**Pull request by BhavyaSheth22**
Added basic ejs template which displays otp. 

TODO: Improve template and make it look good.

### CODE REVIEW

File: server/src/utility/mailer.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    });
+
+    const otpHtml = await ejs.renderFile(__dirname + "/../views/otp.ejs", {
+      otp: otp
+    });
+    // send mail with defined transport object
+    const info = await transporter.sendMail({
+      from: "pride.education.website@gmail.com", // sender address
+      to: email, // list of receivers
+      subject: "OTP verification", // Subject line
```
Comment (ShubhankarKG) [inline server/src/utility/mailer.ts:25]: Nit: Can be updated to "Pride Education | OTP Verification"

File: server/src/utility/mailer.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      html: otpHtml // html body
+    });
+
+    console.log(`Message sent: ${info.messageId}`);
+    return `Message sent: ${info.messageId}`;
```
Comment (ShubhankarKG) [inline server/src/utility/mailer.ts:30]: I don't think we need the return statement here. If this is not required, we can maybe remove it.

**Review by ShubhankarKG — COMMENTED** [review]
Nothing non blocking, feel free to update this as you deem fit.

**Comment by BhavyaSheth22** [comment]
Changes have been done.

**Pull request merged**